mysqltest/config.go

```diff
@@ -68,6 +72,15 @@ func WithVersion(v string) Option {
 	}
 }
 
+// WithLogger overrides the default logger. This logger will receive messages about MySQL startup
+// progress. The default is to use the go "log" package.
+func WithLogger(l Logger) Option {
```

Comment:
No action, but you should check out jba's slog proposal on the Go repo if you're interested for structured logging in the stdlib.

Reply:
Thanks, I hadn't seen that but I welcome it. https://github.com/golang/go/discussions/54763 for posterity.